### DISCUSSION TITLE
Is it possible to override files in the layouts directory with my own?

### DISCUSSION THREAD

**Discussion by kedarjoyner** [jpanther/congo#465] [Q&A] [answered]
I just started using Hugo and I'm learning with Congo as my theme! I am experimenting with [overriding different layout files](https://jpanther.github.io/congo/docs/advanced-customisation/#hugo-project-structure), but not having the results I expected. 

If I wanted to override the `layouts/partial/footer.html` file, I thought creating `layouts/_default/partial/footer.html` would replace the theme file. What's the most sustainable way to do something like this? 

Thanks for any and all help 💕

**Comment by kedarjoyner** [comment]
Looks like I'm unable to delete this discussion, but I think I figured out a workaround by adding my own partials when using a custom homepage. This all chalks up to me being a Hugo n00b: https://jpanther.github.io/congo/docs/homepage-layout/

**Comment by jpanther** [comment, marked as answer]
To override the layouts you simply place them in the same folder hierarchy as the theme, but in your local project folder. So to take your example, to override `theme/layouts/partial/footer.html` you would simply create `your_project/layouts/partial/footer.html` and it will use your template in preference to the one in the theme. You only need to include the `_default` directory when you are overriding layouts that normally reside in that directory.

**Reply by kedarjoyner** [reply]
Awesome! Thank you. Hugo's documentation is a little wonky when it comes to putting all these pieces together, so I appreciate your support and Congo's flexible theme 🎉
